### PR TITLE
[EXPERIMENT] Comparison MassiveOperationScenario between James and Cyrus

### PR DESCRIPTION
**Pull request by quantranhong1999**

### CODE REVIEW

**Comment by quantranhong1999** [comment]
first run:
- james: ERROR c.l.g.i.protocol.command.MoveHandler - MoveHandler command failed
- Cyrus: fail from createMailbox

File: src/it/scala-2.12/com/linagora/gatling/imap/scenario/it/ImapAuthenticationScenarioIT.scala [hunk 1]
```diff
@@ -26,151 +26,158 @@ abstract class BaseIt(server: RunningServer) extends GatlingFunSpec {
   }
 }
 
-
-class ImapAuthenticationScenarioIT extends BaseIt(CyrusServer.start()) {
```
Comment (Arsnael) [inline src/it/scala-2.12/com/linagora/gatling/imap/scenario/it/ImapAuthenticationScenarioIT.scala:30]: You don't need to comment all scenarios to just run one test. You can use in sbt:

```
GatlingIt/testOnly FQDN
```

with I guess your FQDN being here `com.linagora.gatling.imap.scenario.MassiveOperationScenario` :)

Comment (Arsnael) [reply src/it/scala-2.12/com/linagora/gatling/imap/scenario/it/ImapAuthenticationScenarioIT.scala:30]: ah or is it for the CI?

Comment (quantranhong1999) [reply src/it/scala-2.12/com/linagora/gatling/imap/scenario/it/ImapAuthenticationScenarioIT.scala:30]: Yeh, only for the CI

**Comment by quantranhong1999** [comment]
Cyrus:
```
---- Global Information --------------------------------------------------------
> request count                                      22017 (OK=22017  KO=0     )
> min response time                                      6 (OK=6      KO=-     )
> max response time                                  21034 (OK=21034  KO=-     )
> mean response time                                    17 (OK=17     KO=-     )
> std deviation                                        142 (OK=142    KO=-     )
> response time 50th percentile                         16 (OK=16     KO=-     )
> response time 75th percentile                         19 (OK=19     KO=-     )
> response time 95th percentile                         22 (OK=22     KO=-     )
> response time 99th percentile                         24 (OK=24     KO=-     )
> mean requests/sec                                 43.685 (OK=43.685 KO=-     )
---- Response Time Distribution ------------------------------------------------
> t < 800 ms                                         22015 (100%)
> 800 ms < t < 1200 ms                                   0 (  0%)
> t > 1200 ms                                            2 (  0%)
> failed                                                 0 (  0%)
================================================================================

Reports generated in 1s.
Please open the following file: /home/jenkins/build/workspace/Gatling_Imap_build_PR-66/target/gatling-it/imapmassiveoperationscenarioit-20220510080626622/index.html
```

James memory: 
`ERROR c.l.g.i.p.command.ExpungeHandler - ExpungeHandler command failed` (timeout)

**Comment by quantranhong1999** [comment]
```
================================================================================

---- Global Information --------------------------------------------------------

> request count                                      22017 (OK=22017  KO=0     )

> min response time                                      9 (OK=9      KO=-     )

> max response time                                  92782 (OK=92782  KO=-     )

> mean response time                                    36 (OK=36     KO=-     )

> std deviation                                        625 (OK=625    KO=-     )

> response time 50th percentile                         30 (OK=30     KO=-     )

> response time 75th percentile                         31 (OK=31     KO=-     )

> response time 95th percentile                         40 (OK=40     KO=-     )

> response time 99th percentile                         57 (OK=57     KO=-     )

> mean requests/sec                                  23.88 (OK=23.88  KO=-     )

---- Response Time Distribution ------------------------------------------------

> t < 800 ms                                         22015 (100%)

> 800 ms < t < 1200 ms                                   0 (  0%)

> t > 1200 ms                                            2 (  0%)

> failed                                                 0 (  0%)

================================================================================



Reports generated in 0s.

Please open the following file: /home/jenkins/build/workspace/Gatling_Imap_build_PR-66/target/gatling-it/imapmassiveoperationscenarioit-20220511035606401/index.html
```

Seems Cyrus is affected by message size.

**Comment by chibenwa** [comment]
Ok, we experimented, time to close this?

**Pull request closed without merging**